loopchain/blockchain/blocks/v0_4/block_builder.py

```diff
@@ -1,12 +1,8 @@
-from typing import List
-
-from loopchain.blockchain.blocks import BlockBuilder as BaseBlockBuilder, BlockProverType
 from loopchain.blockchain.blocks.block import NextRepsChangeReason
 from loopchain.blockchain.blocks.v0_3 import BlockBuilder
```

Comment:
How about this:
from loopchain.blockchain.blocks.v0_3 import BlockBuilder as BaseBlockBuilder